Modules/Core/Common/include/itkNumberToString.h

```diff
@@ -45,7 +45,6 @@ class ITK_TEMPLATE_EXPORT NumberToString
   std::string
   operator()(TValue val);
 
-private:
```

Comment:
This should perhaps be upgraded to `ITK_DISALLOW_COPY_AND_ASSIGN`.

Reply:
@dzenanz Thanks, but actually I think this class should follow the "rule of zero" and *NOT* delete operator=.  This class has no member functions and simply acts as a thin wrapper around the double conversion functions.

Reply:
That would be even better. So assignment operator's deletion should also be removed?

Reply:
Yes!  I think that was originally a mistake.

Reply:
Code review = find bugs :).